addons/web/static/src/views/list/list_renderer.js

```diff
@@ -1015,6 +1013,13 @@ export class ListRenderer extends Component {
         }
     }
 
+    onCellKeyup(ev) {
+        const hotkey = getActiveHotkey(ev);
+        if (hotkey === "shift") {
+            this.firstArrowShiftRecord = undefined;
+        }
+    }
+
```

Comment:
if you do what I suggested in the comment for the checkbox file, this should be moved within the same listener